### PR TITLE
Fixed a few small things in the sample_package

### PR DESCRIPTION
**Pull request by garethellis0**

### CODE REVIEW

File: src/sample_package/src/MyNode.cpp [hunk 1]
```diff
@@ -15,18 +15,19 @@ MyClass::MyClass(int argc, char **argv, std::string node_name) {
...
     std::string topic_to_subscribe_to = "subscribe_topic";
-    int refresh_rate = 10;
-    my_subscriber = nh.subscribe(topic_to_subscribe_to, refresh_rate, &MyClass::subscriberCallBack, this);
+    int queue_size = 10;
+    my_subscriber = nh.subscribe(topic_to_subscribe_to, queue_size, &MyClass::subscriberCallBack, this);
 
     // Setup Publisher(s)
     std::string topic = private_nh.resolveName("publish_topic");
-    uint32_t queue_size = 1;
+    queue_size = 10;
```
Comment (RobynCastro) [inline src/sample_package/src/MyNode.cpp]: isn't this a redundant line? but otherwise looks good!

Comment (garethellis0) [reply src/sample_package/src/MyNode.cpp]: It is redundant, but I thought it best to set it again here so people don't get the idea that the queue sizes need to be the same. I'll change the value so it's a bit more clear

**Comment by garethellis0** [comment]
In this case the queue size really doesn't matter. But in general: https://answers.ros.org/question/190357/rospypublisher-whats-a-good-choice-for-queue_size/

**Comment by garethellis0** [comment]
@RobynCastro - ready for re-review

**Pull request merged**